Assets/Code/src/Runtime/Match/Components/MatchCountdownController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace HouraiTeahouse.FantasyCrescendo.Matches
+{
+
+public class MatchCountdownController : MonoBehaviour
+{
+    public Text TextUI;
+    public float GoDuration = 1.25f;
```

Comment:
We are aiming to play some audio while this counts down. Potential option: take in a array of AudioClips, play them to an AudioSource and use that as the base for how long to wait. Should also avoid needing to hard-code the 3 second timer.